drivers/include/lcd.h

```diff
@@ -70,18 +87,58 @@ extern "C" {
+ *       are enabled at the same time. In this case the interface parameters
+ *       for the SPI serial interface and the MCU 8080 parallel 8-/16-bit
+ *       interfaces are defined. @ref lcd_params_t::spi must then be set to
+ *       @ref SPI_UNDEF for displays that use the MCU-8080-parallel-8-/16-bit
+ *       interface, i.e. @ref lcd_params_t::spi is then used to detect the
+ *       interface mode.
  */
 typedef  struct {
+#if MODULE_LCD_SPI || DOXYGEN
+    /* Interface parameters used for serial interface */
     spi_t spi;                  /**< SPI device that the display is connected to */
     spi_clk_t spi_clk;          /**< SPI clock speed to use */
     spi_mode_t spi_mode;        /**< SPI mode */
+#endif
+#if MODULE_LCD_PARALLEL || DOXYGEN
+    /* Interface parameters used for MCU 8080 8-bit parallel interface */
+    gpio_t wrx_pin;             /**< pin connected to the WRITE ENABLE line */
+    gpio_t rdx_pin;             /**< pin connected to the READ ENABLE line */
+    gpio_t d0_pin;              /**< pin connected to the D0 line */
+    gpio_t d1_pin;              /**< pin connected to the D1 line */
+    gpio_t d2_pin;              /**< pin connected to the D2 line */
+    gpio_t d3_pin;              /**< pin connected to the D3 line */
+    gpio_t d4_pin;              /**< pin connected to the D4 line */
+    gpio_t d5_pin;              /**< pin connected to the D5 line */
+    gpio_t d6_pin;              /**< pin connected to the D6 line */
+    gpio_t d7_pin;              /**< pin connected to the D7 line */
+#if MODULE_LCD_PARALLEL_16BIT || DOXYGEN
+    /* Interface parameters used for MCU 8080 16-bit parallel interface */
+    gpio_t d8_pin;              /**< pin connected to the D8 line */
+    gpio_t d9_pin;              /**< pin connected to the D9 line */
+    gpio_t d10_pin;             /**< pin connected to the D10 line */
+    gpio_t d11_pin;             /**< pin connected to the D11 line */
+    gpio_t d12_pin;             /**< pin connected to the D12 line */
+    gpio_t d13_pin;             /**< pin connected to the D13 line */
+    gpio_t d14_pin;             /**< pin connected to the D14 line */
+    gpio_t d15_pin;             /**< pin connected to the D15 line */
+#endif /* MODULE_LCD_PARALLEL_16BIT */
+#endif /* MODULE_LCD_PARALLEL */
+    /* Common interface parameters */
     gpio_t cs_pin;              /**< pin connected to the CHIP SELECT line */
     gpio_t dcx_pin;             /**< pin connected to the DC line */
-    gpio_t rst_pin;             /**< pin connected to the reset line */
-    bool rgb;                   /**< True when display is connected in RGB mode
-                                 *  False when display is connected in BGR mode */
+    gpio_t rst_pin;             /**< pin connected to the RESET line */
+    bool rgb;                   /**< True when display is connected in RGB mode\n
+                                     False when display is connected in BGR mode */
```

Comment:
Other modes than RGB and BGR? `adafruit-pybadge`, `esp32s3-usb-otg` and `esp32s2-lilygo-ttgo-t8` use RGB mode, `sipeed-longan-nano` uses the BGR mode.

Reply:
I guess turning this into an

```C
typedef enum {
    LCD_MODE_BGR = 0,
    LCD_MODE_RBG,
} lcd_mode_t;
```

would be simple enough if the need arises later. (although I think it would make the code more obvious already)

But I never see `->rgb` being used (or the RBG boards setting it to `true` for that matter)

Reply:
`->rgb` is not new, it was always defined as a Boolean in the driver. I have only modified the comment. It is used in device drivers, e.g. in https://github.com/RIOT-OS/RIOT/blob/9609a49da754092995289509e51caa07af22db9b/drivers/st77xx/st77xx.c#L91